.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+# Python CircleCI 2.0 configuration file
+#
+version: 2
+
+jobs:
+  build:
+    working_directory: ~/ingest-client
+    docker:
+      - image: circleci/python:3.6.5
+
+    steps:
+        - checkout
+        
+        - run: mkdir test-reports
+        
+        - restore_cache:  # ensure this step occurs *before* installing dependencies
+            keys:
+              - cache-{{ checksum "requirements.txt" }}
+              - cache-
+                
+        - run:
+            name: Installing pip requirements at top level
+            command: |
+              sudo pip install -r requirements.txt
+                
+        - run:
+            name: Installing pip requirements at plugins level
+            command: |
+              cd ingestclient/plugins/requirements/
+              sudo pip install -r cloudvolume_requirements.txt
+                
+        - save_cache:
+            paths:
+              - ~/.local
+              - ~/.cache
+            key: cache-{{ checksum "requirements.txt" }}
+
+        ####  Run Tests
+
+        - run:
+            name: Setup Dummy AWS creds
+            command: |
+              mkdir ~/.aws
+              echo -e "[default]\naws_access_key_id = foo\naws_secret_access_key = bar" > ~/.aws/credentials
+              echo -e "[default]\nregion = us-east-1\noutput = json" > ~/.aws/config        
+
+        - run:
+            name: Running tests ... python unit tests [pipe stdout/err to results.txt]
+            command: |
+              python -m unittest -v  &> /tmp/unittest_results.txt
+
+        - run:
```

Comment:
Let's remove the use of nose. We don't need to do both.

Reply:
Sure thing ... removed the use of nose. Now only running python unit tests for both py2 and py3.  Furthermore, added .vscode/ to .gitignore to avoid user's debugger and runtime settings from being checked into git.  
The latest workflow results can be seen online in circle: 
    https://circleci.com/workflow-run/72252e42-ecc1-456d-b810-d8e9d9d585f2
The workflow results should show both py2 and py3 jobs.  Just click on either of them to review the test run and results ...  Both passed as of today's status. Thanks.